internal/command/apply.go

```diff
@@ -385,6 +385,15 @@ Options:
   -state-out=path        Path to write state to that is different than
                          "-state". This can be used to preserve the old
                          state.
+                         
+  -var 'foo=bar'         Set a value for one of the input variables in the root
+                         module of the configuration. Use this option more than
+                         once to set more than one variable.
+
+  -var-file=filename     Load variable values from the given file, in addition
+                         to the default files terraform.tfvars and *.auto.tfvars.
+                         Use this option more than once to include more than one
+                         variables file.
```

Comment:
> If you don't provide a saved plan file

But this is also for when you provide a saved plan 🤔  I would think we need to reconsider the wording, because some of them are now supported and some are not when passing a planfile 🙈